content/tutorial/02-sveltekit/03-loading-data/03-universal-load-functions/README.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Universal load functions
+---
+
+> Coming soon
```

Comment:
This feels rather drastic, and a "coming soon" so early is no good first impression in my opinion. `+page/layout.js` is a pretty big part of SvelteKit, so not teaching it ... I don't know, that feels wrong. I think it would be better to keep it, or adjust the text and say "let's assume that this is an external API".

Reply:
yeah, i intend to follow up with stackblitz ASAP about why `fetch` is failing in this context. but having magic hidden routes is just a huge no-no in my opinion